### ISSUE TITLE
[question] are these tight  with the powerpack?

### ISSUE THREAD

**Issue by kirk86** [msoedov/Alfred-collection#2]
Hi, I presume that none of these workflows can be tested if u don't have the powerpack installed, right?

**Comment by pjattke** [comment]
Yes, workflows are an Alfred PowerPack feature only. Unfortunately, there is also no free Alfred PowerPack trial available.

**Comment by youtextme** [comment]
Yes — Alfred workflows are Powerpack-only, and there's still no free Powerpack trial, so you can't really "try the collection" on Alfred free.

If you're deciding whether to buy Powerpack mainly to unlock workflows: free Raycast is worth a dual-run first (Option-Space alongside Alfred). The Store covers a lot of the same jobs (clipboard, snippets, apps, many workflow equivalents) before you commit to a paid Alfred license.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.
